src/Test/nlp.jl

```diff
@@ -172,6 +200,7 @@ end
 
 hs071_test(model, config) = hs071test_template(model, config, HS071(true))
 hs071_no_hessian_test(model, config) = hs071test_template(model, config, HS071(false))
+hs071_hessian_vector_product_test(model, config) = hs071test_template(model, config, HS071(false, true))
```

Comment:
Why not add it to the `@moitestset` ?

Reply:
I thought that as Ipopt does not support Hessian vector product, it would be better to let `hs071_hessian_vector_product_test` out of the `@moitestset`.